src/hooks/validation.ts

```diff
@@ -38,21 +39,23 @@ type QueryParamValidators<ReqParams> = {
 }
 
 const resourceSearchValidators: QueryParamValidators<ResourceSearchRequest> = {
-  resource_type:  withinEnum(Object.values(ResourceTypeEnum)),
-  department:     withinEnum(Object.values(DepartmentEnum)),
-  level:          withinEnum(Object.values(LevelEnum)),
-  platform:       withinEnum(Object.values(PlatformEnum)),
-  offered_by:     withinEnum(Object.values(OfferedByEnum)),
-  sortby:         values => withinEnum(Object.values(SortByEnum))(values)[0],
-  q:              first,
-  topic:          identity,
-  certification:  firstBoolean,
-  professional:   firstBoolean,
-  aggregations:   withinEnum(Object.values(AggregationsEnum)),
-  course_feature: identity,
-  limit:          firstNumber,
-  offset:         firstNumber,
-  id:             numbers
+  resource_type:   withinEnum(Object.values(ResourceTypeEnum)),
+  department:      withinEnum(Object.values(DepartmentEnum)),
+  level:           withinEnum(Object.values(LevelEnum)),
+  platform:        withinEnum(Object.values(PlatformEnum)),
+  offered_by:      withinEnum(Object.values(OfferedByEnum)),
+  sortby:          values => withinEnum(Object.values(SortByEnum))(values)[0],
+  q:               first,
+  topic:           identity,
+  certification:   firstBoolean,
+  professional:    firstBoolean,
+  aggregations:    withinEnum(Object.values(AggregationsEnum)),
+  course_feature:  identity,
+  limit:           firstNumber,
+  offset:          firstNumber,
+  id:              numbers,
+  free:            firstBoolean,
+  learning_format: withinEnum(Object.values(LearningFormatEnum))
```

Comment:
Needed to make this update here because otherwise typecheck is unhappy about LearningResourcesSearchApiLearningResourcesSearchRetrieveRequest having extra parameters